lib/pivotal-tracker/me.rb

```diff
@@ -0,0 +1,12 @@
+class PivotalTracker::Me
```

Comment:
Please inherit from our new `PivotalTracker::Resource` class (introduced as part of https://github.com/jasonnoble/pivotal-tracker/pull/11/files#diff-db66140f03dd10afa01447b572c04b2dR1)